homeassistant/components/modbus/light.py

```diff
@@ -0,0 +1,139 @@
+"""Support for Modbus lights."""
```

Comment:
Please do one platform per PR.

Reply:
Sure. What about this PR. Should I close it and start new ones?

Reply:
It's ok to keep the switch changes if you like.

Reply:
OK, I will keep the switch changes in this PR. For clarity, I will rename the title and description.
Then, I will create separate PRs for Fan and Light, where I will reference this PR.

Main discussion can still be here. I see others are commenting with suggested features. It is good to have it in a single place.